eml/components/exercise/LeaveButton.js

```diff
@@ -20,17 +20,17 @@ const LeaveButton = ({ navigationPlace, courseId }) => {
       }
       icon={
         <Icon
-          size={40}
+          size={25}
           name="chevron-left"
           type="material-community"
-          color="rgb(75,85,99)"
+          color="#383838"
```

Comment:
It is necessary to use stylesheets due to some reason?

If not, then integration team has agreed that all use NativeWind.